src/main/java/org/opensearch/security/support/ConfigConstants.java

```diff
@@ -79,6 +79,7 @@ public class ConfigConstants {
     public static final String OPENDISTRO_SECURITY_MASKED_FIELD_CCS = OPENDISTRO_SECURITY_CONFIG_PREFIX + "masked_fields_ccs";
 
     public static final String OPENDISTRO_SECURITY_CONF_REQUEST_HEADER = OPENDISTRO_SECURITY_CONFIG_PREFIX + "conf_request";
+    public static final String OPENDISTRO_SECURITY_REQUEST_HEADERS = OPENDISTRO_SECURITY_CONFIG_PREFIX + "request_headers";
```

Comment:
nit: can the new variable use opensearch instead of opendistro?

Reply:
Renamed to OpenSearch